test/elixir/test/bulk_docs_test.exs

```diff
@@ -0,0 +1,147 @@
+defmodule BulkDocsTest do
+  use CouchTestCase
+
+  @moduletag :bulk_docs
+
+  @moduledoc """
+  Test CouchDB bulk docs
+  This is a port of bulk_docs.js
+  """
+
+  @doc_count 5
+
+  @tag :with_db
+  test "bulk docs can create, update, & delete many docs per request", ctx do
+    db = ctx[:db_name]
+    docs = make_docs(@doc_count)
+    resp = bulk_post(docs, db)
+    assert revs_start_with(resp.body, "1-")
+    docs = revise(docs, resp.body)
```

Comment:
Naming for `revise` is unfortunate. It is not clear what the function does without looking at its source. Should we name it `to_map`?

Reply:
`to_map` is bad as well since the function do something else.

Reply:
Hmm, `revise` seems like a decent name (to me), since it takes the value of the "rev" field returned from the server and inserts or updates it as `:_rev` into the doc (map).